src/webext/sendMessageToBackground.ts

```diff
@@ -1,4 +1,8 @@
+/* eslint-disable @typescript-eslint/ban-ts-ignore */
+import { serialize } from './serializer';
+
 const sendMessageToBackground = (message: unknown) =>
-  browser.runtime.sendMessage(message);
+  // @ts-ignore
+  browser.runtime.sendMessage(serialize(message));
```

Comment:
Serialization is done automatically if a serializable object is passed … 
https://developer.mozilla.org/fr/docs/Mozilla/Add-ons/WebExtensions/API/runtime/sendMessage
Can you explain why we’d do this manually ?

Reply:
See b6fa7ad body and #904